docs/reference/ml/trained-models/apis/start-trained-model-deployment.asciidoc

```diff
@@ -25,6 +25,11 @@ Currently only `pytorch` models are supported for deployment. Once deployed
 the model can be used by the <<inference-processor,{infer-cap} processor>>
 in an ingest pipeline or directly in the <<infer-trained-model>> API.
 
+A model can be deployed multiple times by using deployment IDs. A deployment ID 
+must be unique and should not match any other deployment ID or model ID, unless 
+it is the same as the ID of the model being deployed. If `deployment_id` is not 
+set, it defaults to the `model_id`.
```

Comment:
It would also be good to include a paragraph similar to this in https://www.elastic.co/guide/en/machine-learning/8.8/ml-nlp-deploy-models.html#ml-nlp-deploy-model

Maybe not exactly the same, as it might not be appropriate to mention specific argument names, but that section in the higher level docs should explain that models can be deployed multiple times. It could also explain that the likely reason for wanting to do this is so that the speed of searches is not affected by ingest volume and the speed of ingest is not affected by search volume. Deploying a model twice and using one deployment for search and another for ingest avoids the danger of hard-to-diagnose performance problems caused by interactions between the two.

Reply:
That's up next. Those docs should also mention that search requests are prioritised over ingest requests

Reply:
It's on my TODO list. Those docs live in a different repo, so another PR is needed.